### PR TITLE
meta-isar: remove name resolution message during sudo

### PR DESCRIPTION
**Pull request by bhargavdas**
Signed-off-by: Bhargav Das <bhargav_das@mentor.com>

### CODE REVIEW

File: meta-isar/recipes-conf/hostname/files/postinst [hunk 1]
```diff
@@ -3,3 +3,8 @@
 set -e
 
 echo "mtda" > /etc/hostname
+echo "127.0.0.1    localhost\n/
+    127.0.1.1    mtda\n/
+    ::1    localhost ip6-localhost ip6-loopback\n/
+    ff02::1    ip6-allnodes\n/
+    ff02::2    ip6-allrouters" >> /etc/hosts
```
Comment (chombourger) [inline meta-isar/recipes-conf/hostname/files/postinst:10]: can you check if adding the `netbase` package to the image would help?
see /var/lib/dpkg/info/netbase.postinst - it is supposed to create /etc/hosts

File: meta-isar/recipes-conf/hostname/files/postinst [hunk 2]
```diff
@@ -3,3 +3,8 @@
 set -e
 
 echo "mtda" > /etc/hostname
+echo "127.0.0.1    localhost\n/
```
Comment (vj-kumar) [inline meta-isar/recipes-conf/hostname/files/postinst:6]: I am not sure why you need \n/ at the end.

I think the below should be sufficient.
echo "127.0.0.1    localhost
127.0.1.1    mtda
::1    localhost ip6-localhost ip6-loopback
ff02::1    ip6-allnodes
ff02::2    ip6-allrouters" >> /etc/host

File: meta-isar/recipes-conf/hostname/files/postinst [hunk 3]
```diff
@@ -3,3 +3,10 @@
 set -e
 
 echo "mtda" > /etc/hostname
+
+# add hostname in /etc/hosts
+echo "127.0.0.1    localhost
+    127.0.1.1    mtda
```
Comment (chombourger) [inline meta-isar/recipes-conf/hostname/files/postinst:9]: this is the only line we need to add

**Comment by chombourger** [comment]
add DEBIAN_DEPENDS = "netbase" to the recipe (`.bb` file)

File: meta-isar/recipes-conf/hostname/files/postinst (hunk 3 above)
Comment (bhargavdas) [reply meta-isar/recipes-conf/hostname/files/postinst:9]: I tried adding only `echo "127.0.1.1    mtda" >> /etc/hosts` but it overrides the other contents the file. That's I added all the lines

Comment (vj-kumar) [reply meta-isar/recipes-conf/hostname/files/postinst:9]: >> should basically append. Just tried it to confirm.

**Comment by bhargavdas** [comment]
> add DEBIAN_DEPENDS = "netbase" to the recipe (`.bb` file)

getting the following error 
`dpkg-gencontrol: warning: can't parse dependency base-files netbase`
after adding 
```
DEBIAN_DEPENDS = "base-files netbase"
```

**Comment by chombourger** [comment]
> > add DEBIAN_DEPENDS = "netbase" to the recipe (`.bb` file)
> 
> getting the following error `dpkg-gencontrol: warning: can't parse dependency base-files netbase` after adding
> 
> ```
> DEBIAN_DEPENDS = "base-files netbase"
> ```
DEBIAN_DEPENDS = "base-files, netbase"

(entries are comma-separated in debian/control)

File: meta-isar/recipes-conf/hostname/files/postinst (hunk 3 above)
Comment (chombourger) [reply meta-isar/recipes-conf/hostname/files/postinst:9]: > I tried adding only `echo "127.0.1.1 mtda" >> /etc/hosts` but it overrides the other contents the file. That's why I added all the lines

even with DEBIAN_DEPENDS correctly set?

Comment (bhargavdas) [reply meta-isar/recipes-conf/hostname/files/postinst:9]: that I have not checked.checking now

Comment (bhargavdas) [reply meta-isar/recipes-conf/hostname/files/postinst]: it is appending after adding netbase package.

File: meta-isar/recipes-conf/hostname/files/postinst [hunk 4]
```diff
@@ -3,3 +3,6 @@
 set -e
 
 echo "mtda" > /etc/hostname
+
+# add hostname in /etc/hosts
```
Comment (chombourger) [inline meta-isar/recipes-conf/hostname/files/postinst:7]: add hostname to /etc/hosts

**Comment by chombourger** [comment]
let's squash the 2 commits (git rebase -i origin/master and use `squash` instead of `pick` for the second commit)
let's rename the commit to: `meta-isar/recipes-conf: add "mtda" to /etc/hosts`

**Review by chombourger — APPROVED** [review]

**Comment by chombourger** [comment]
Thanks @bhargavdas for your contribution!

**Pull request merged**